crates/uv-cli/src/lib.rs

```diff
@@ -4759,6 +4759,10 @@ pub struct PythonFindArgs {
         conflicts_with = "no_system"
     )]
     pub script: Option<PathBuf>,
+
+    /// Show the Python version that would be used instead of the path to the interpreter.
+    #[arg(long)]
+    pub show_version: bool,
```

Comment:
Or `--version`? (Defer to you.)

Reply:
Yeah I have a preference for `--version` but it's taken.

```
❯ uv python find --version
uv-python-find 0.6.9 (3d9460278 2025-03-20)
```

Do you think we should override it like that?